File: 5 - Coding Interview Workshops/Week7/CodeFiles/JavaScript/recursionIterative.js
let students = ["Zahra", "Moumita", "Anri", "Julien"];

function passNote (destination){
    let found = False;
    i = 0;

    while (!found){
        if (i == len(students))
            return "Destination not found";
        else if (destination == students[i]){
            found = True;
            return "Note delivered";
        }
        
        
        i += 1;
    }
}

File: 5 - Coding Interview Workshops/Week07/CodeFiles/Python/recursionIterative.py
students = ["Zahra", "Moumita", "Anri", "Julien"]

def passNote (destination):
    found = False
    i = 0

    while (not found):
        if (i == len(students)):
            return "Destination not found"
        elif (destination == students[i]):
            found = True
            return "Note delivered"
        
        i += 1




